absl/base/internal/unscaledcycleclock.cc

```diff
@@ -57,11 +63,40 @@ double UnscaledCycleClock::Frequency() {
 #elif defined(__powerpc__) || defined(__ppc__)
 
 int64_t UnscaledCycleClock::Now() {
+#ifdef __GLIBC__
   return __ppc_get_timebase();
+#elif defined(__FreeBSD__)
```

Comment:
Is this code portable? I think it will work on non-FreeBSD ppc platforms as well. Should this just be `#else`?